libs/elements/layout/buttons/src/lib/components/elewa-normal-button/elewa-normal-button.component.html

```diff
@@ -0,0 +1,8 @@
+<!-- The button element and how it's rendered in the component's template -->
+<button 
+    class="elewa-normal-button"
+    [style.backgroundColor]="buttonData.bgColor"
+    [style.color]="buttonData.color"
+>
+    <ng-content></ng-content>
+</button>
```

Comment:
Instead of using ng-content, you can pass the data as input to the button component. Since your already doing that in the ts file. Use interpolation to showcase that data.